pkg/sysregistriesv2/system_registries_v2.go

```diff
@@ -570,6 +579,66 @@ func UnqualifiedSearchRegistries(ctx *types.SystemContext) ([]string, error) {
 	return config.UnqualifiedSearchRegistries, nil
 }
 
+// GetCredsStore return the list of credetntial store configed by creds_store for registries
+func GetCredsStore(ctx *types.SystemContext) ([]string, error) {
+	config, err := getConfig(ctx)
+	if err != nil {
+		return nil, err
+	}
+	return config.CredsStore, nil
+}
+
+// GetCredConfigs returns a slice of credential stores configred in registries.conf
+// caller should use GetRegistryCredentialHelpers to get cred helper of specified registry
+func GetCredConfigs(sys *types.SystemContext) ([]string, error) {
+	var (
+		credConfigs []string
+		credHelpers = make(map[string]bool)
```

Comment:
Non-blocking? The naming distinction between `credConfigs` and `credHelpers`, both of which reference the same data, does not really make sense, especially when there isn’t any other kind of configuration. `helperList` and `helperSet` would at least be descriptive. Or it might be good enough to just have `helpers` (the set) and a very short-lived `res`/`result` variable declared only for the last loop — the shorter the scope of a symbol, the less descriptive/distinctive the name needs to be.